superset/views/base.py

```diff
@@ -365,6 +365,17 @@ def common_bootstrap_payload() -> Dict[str, Any]:
         "extra_sequential_color_schemes": conf["EXTRA_SEQUENTIAL_COLOR_SCHEMES"],
         "extra_categorical_color_schemes": conf["EXTRA_CATEGORICAL_COLOR_SCHEMES"],
         "theme_overrides": conf["THEME_OVERRIDES"],
+        "allowed_extensions": {
+            "excel_extensions": bool(
+                conf["EXCEL_EXTENSIONS"].intersection(conf["ALLOWED_EXTENSIONS"])
+            ),
+            "csv_extensions": bool(
+                conf["CSV_EXTENSIONS"].intersection(conf["ALLOWED_EXTENSIONS"])
+            ),
+            "columnar_extensions": bool(
+                conf["COLUMNAR_EXTENSIONS"].intersection(conf["ALLOWED_EXTENSIONS"])
+            ),
+        },
```

Comment:
to dry this up a bit more, you should be able to add these four conf keys to `FRONTEND_CONF_KEYS` and they'll get passed through the conf param.

Reply:
I agree with elizabeth lets just add these keys to `frontend_config` object